Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/Booking.hs

```diff
@@ -69,6 +70,7 @@ data Booking = Booking
     estimatedDuration :: Seconds,
     fareParams :: FareParameters,
     riderName :: Maybe Text,
+    paymentMethodId :: Maybe (Id DMPM.MerchantPaymentMethod),
```

Comment:
That way with every `MerchantPaymentMethod` change, info about every booking will be changed as well. Should we store whole `MerchantPaymentMethod` here instead of id ref?

Reply:
> @DonDokidon I don't think that `MerchantPaymentMethod` will be changed somehow, It's better to create new `MerchantPaymentMethod`, than change existing one. The only thing, that can be changed here is `priority`. Also we can add some `enabled` flag for switching off some `MerchantPaymentMethod`

Why do you have `updatedAt` field in it then?